frontend/src/metabase/query_builder/components/view/ViewButton.tsx

```diff
@@ -31,6 +32,10 @@ const ViewButton = styled(Button)<Props>`
   > .Icon {
     opacity: 0.6;
   }
+
+  .Icon + div {
```

Comment:
to not target some unrelated diff maybe:
```
.Icon:not(:last-child) {
  margin-right: 0.25rem;
}
```

Reply:
we need to override styles of the sibling, that's why I used `.Icon + div`